supportedBackends/tes/src/main/scala/cromwell/backend/impl/tes/TesExpressionFunctions.scala

```diff
@@ -3,10 +3,24 @@ package cromwell.backend.impl.tes
 import cromwell.backend.sfs.SharedFileSystemExpressionFunctions
 import cromwell.backend.standard.StandardExpressionFunctionsParams
 
+import scala.concurrent.Future
+
 class TesExpressionFunctions(standardParams: StandardExpressionFunctionsParams) extends SharedFileSystemExpressionFunctions(standardParams) {
+  override lazy implicit val ec = standardParams.executionContext;
 
   override def preMapping(str: String) = {
     if (str.startsWith("/") || str.startsWith("ftp://")) str
     else standardParams.callContext.root.resolve(str).toString
   }
+
+  override def glob(pattern: String): Future[Seq[String]] = {
+    import wom.values.GlobFunctions._
+    val globPatternName = globName(pattern)
+    val listFilePath = callContext.root.resolve(s"${globName(pattern)}.list")
+    asyncIo.readLinesAsync(listFilePath.toAbsolutePath) map { lines =>
```

Comment:
Have we discovered here that calling `toRealPath` on a blob file always throws an error? If so, could we override that method on `BlobPath` rather than changing it here?